server/service/http/api.go

```diff
@@ -144,3 +146,35 @@ func (a *API) transferLeader(writer http.ResponseWriter, req *http.Request) {
 
 	a.respond(writer, "ok")
 }
+
+type RouteRequest struct {
+	ClusterName string `json:"clusterName"`
+	SchemaName  string `json:"schemaName"`
+	Table       string `json:"table"`
+}
+
+func (a *API) route(writer http.ResponseWriter, req *http.Request) {
+	var routeRequest RouteRequest
+	err := json.NewDecoder(req.Body).Decode(&routeRequest)
+	if err != nil {
+		log.Error("decode request body failed", zap.Error(err))
```

Comment:
```suggestion
		log.Error("decode request body failed", zap.Error(err),zap.String("body", string(req.Body)))
```

Reply:
Request body has been print in `printRequestInsmt`, so it is not need here.